### PR TITLE
Closes tddbin/katas#53

### PR DESCRIPTION
**Pull request by jizusun**
Though the fix is a little bit tricky, I managed to make it pass, and hope this can help.

image

### CODE REVIEW

**Comment by jizusun** [comment]
Though the fix is a little bit tricky, I managed to make it pass, and hope this can help.

image

File: katas/es6/language/class/super-in-constructor.js [hunk 1]
```diff
@@ -41,14 +41,14 @@ describe('Inside a class`s constructor `super()` can be used', () => {
...
+    class A { constructor() { "parent"; } }
     class B extends A {
       constructor() {
         super();
         //// this.isTop = '' + super.konstructer;
         this.isTop = '' + super.constructor;
       }
     }
-    assert.equal(new B().isTop, 'class A {constructor() {"parent"}}');
+    assert.equal(new B().isTop.replace(/\n\s+/g, ' '), 'class A { constructor() { "parent"; } }');
```
[the remark this answers is not in the document]
Comment (jizusun) [reply katas/es6/language/class/super-in-constructor.js]: Yes, this one is less error-prone.

**Comment by wolframkriesing** [comment]
that's a tough one ... what do you think about this?
```
  it('use `super.constructor` to find out if there is a parent constructor', () => {
    class ParentClassA {constructor() {"parent"}}
    class B extends ParentClassA {
      constructor() {
        super();
        //// this.isTop = '' + super.konstructer;
        this.isTop = '' + super.constructor;
      }
    }
    assert(new B().isTop.includes('ParentClassA'), new B().isTop);
  });
```

**Comment by jizusun** [comment]
Hi sir, I couldn't come up with a better solution than yours 😅

**Pull request merged**